src/docs/commands_keybindings.md

```diff
@@ -1,122 +1,142 @@
 ---
```

Comment:
Should the file be renamed for clarity with `menus` in it?

Reply:
Perhaps this one is not worth it, It'll break anyone that refers to the old link.

Reply:
Yes, i did not rename this on purpose